client/components/forms/form-text-input/index.jsx

```diff
@@ -72,7 +75,7 @@ export default class FormTextInput extends PureComponent {
 
 		return (
 			<input
-				type="text"
+				type={ inputType }
 				{ ...rest }
```

Comment:
Just out of curiosity: setting `type="number"` would not "override" the default "text" as it is passed in `...rest`? Or is the first attribute value the one taken into account?